From: SeongJae Park <sj@kernel.org>
Date: Mon, 12 Jan 2026 17:44:28 -0800
Subject: [PATCH] Docs/mm/damon/design: link repology instead of Fedora package

The document is introducing Fedora as one way to get DAMON user-space
tool (damo).  The user-space tool available on not only Fedora but
multiple Linux distros, though.  Introduce the repology.org page that
shows damo packaging status for multiple Linux distros.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/mm/damon/design.rst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/mm/damon/design.rst b/Documentation/mm/damon/design.rst
index 797cc566fbd5..326e34601e23 100644
--- a/Documentation/mm/damon/design.rst
+++ b/Documentation/mm/damon/design.rst
@@ -888,8 +888,8 @@ The ABIs are designed to be used for user space applications development,
 rather than human beings' fingers.  Human users are recommended to use such
 user space tools.  One such Python-written user space tool is available at
 Github (https://github.com/damonitor/damo), Pypi
-(https://pypistats.org/packages/damo), and Fedora
-(https://packages.fedoraproject.org/pkgs/python-damo/damo/).
+(https://pypistats.org/packages/damo), and multiple distros
+(https://repology.org/project/damo/versions).
 
 Currently, one module for this type, namely 'DAMON sysfs interface' is
 available.  Please refer to the ABI :ref:`doc <sysfs_interface>` for details of
-- 
2.47.3

